composer.json

```diff
@@ -1,5 +1,6 @@
 {
     "require": {
+        "php": "^7.1",
```

Comment:
At the Docker machine, we're working on PHP 7.1. I have no knowledge of what's the minimum PHP version on production, but if it's 7.1, I'd suggest to leave this in composer, so we are more aware of the current minimum version. Since PHPStorm was telling me 7.0 as minimum.

Reply:
Currently the production server is using PHP 7.0. But we are all working with the docker images with the 7.1. Maybe we can wait until we upgrade to 7.1 before setting it in the composer.json.

Reply:
Definitely, will make it PHP 7.0 then.